src/lib/repository/Upsert.ts

```diff
@@ -77,34 +120,93 @@ export class Upsert {
     }
 
     public static async insertData(connection: Connection, records: Array<any>, sObjectName: string) {
```

Comment:
Czytam te zmiany i dochodzę do wniosku że lepiej byłoby mieć jakiś obiekt który to ogarnia, bo tak się robi nam straszne spaghetti. W sensie, żeby funkcja `insertData` wykonywała prostą logikę a cały errorHandling / warningHandling był jakoś wbudowany / wyniesiony gdzieś obok.

Reply:
Dobry pomysł, spróbuję dzisiaj zrobić jakiś grubszy refactor tego. 
Tutaj jest dużo powtórzonego kodu, którego próbowałem powkładać do osobnych metod, ktore można by było wywoływać z różnymi parmetrami, ale kiedy tak zrobiłem wszystko przestało mi działać, pewnie przez asynchroniczne wywołania i ich chainowanie.

Reply:
Zrobiłem klasę MessageHandler która zawiera w sobie większość, jeśli nie całą, komunikację z userem. Póki co zrefactorowałem tylko klasę upsert, ale mogę w przyszłości zająć się też pozostałymi.